src/test/kotlin/uk/gov/dluhc/notificationsapi/testsupport/testdata/models/NotificationDestinationBuilder.kt

```diff
@@ -25,10 +25,8 @@ fun buildMessageAddressPostalAddress(
     address = address
 )
 
-fun buildMessageAddressOverseasAddress(
-    addressee: String? = faker.name().firstName(),
-    address: OverseasAddress = buildOverseasAddressMessage()
-): MessageAddressOverseasAddress = MessageAddressOverseasAddress(
-    addressee = addressee,
+fun buildMessageAddressOverseasElectorAddress(
+    address: OverseasElectorAddress = buildOverseasAddressMessage()
+): MessageAddressOverseasElectorAddress = MessageAddressOverseasElectorAddress(
     address = address
 )
```

Comment:
If we're only setting one field in this builder we may as well scrap it and just do `MessageAddressOverseasElectorAddress(...)` inline